packages/admin-stories/src/admin/mui/Menu.tsx

```diff
@@ -1,11 +1,14 @@
 import { MasterLayout, Menu, MenuCollapsibleItem, MenuItemAnchorLink, MenuItemRouterLink, useWindowSize } from "@comet/admin";
-import { Typography } from "@material-ui/core";
+import { Box, Typography } from "@material-ui/core";
 import { Dashboard, GitHub, Launch, List, Settings } from "@material-ui/icons";
 import { storiesOf } from "@storybook/react";
 import * as React from "react";
 import { Route, Switch } from "react-router";
 import StoryRouter from "storybook-react-router";
 
+// @ts-ignore
```

Comment:
ts-ignore :scream: 

the easiest way to work around this is by creating a tsx out of the svg (similar to icons)